sdk/data-generators/postcard-generator/src/main.rs

```diff
@@ -0,0 +1,167 @@
+//! Data generator sdk/rust-examples/postcard-native and postcard-wasm
+//!
+//! # Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! # Copyright
+//!
+//! See the file `LICENSE_MIT.markdown` in the Veracruz root directory for licensing
+//! and copyright information.
+//!
+//! # Example
+//! ```
+//! cargo run -- --file [STRING] --size [VEC_SIZE] --seed [RANDOM_SEED];
+//! ```
+
+use clap::{App, Arg};
+use rand::{rngs::StdRng, Rng, SeedableRng};
+use serde::{Deserialize, Serialize};
+use std::{error::Error, fs, string::String, vec::Vec};
+
+#[derive(Deserialize, Serialize, Clone, Debug)]
+enum E1 {
```

Comment:
Aren't many of the items in this file duplicates to the items in the `postcards.rs` file? Is there a reason they are repeated here?